### PR TITLE
FIXED issue #20 made searchbar sticky

### PR DESCRIPTION
**Pull request by veerendra19codes**
#FIXED issue #20
Made the searchbar sticky from top so that user can search while scrolling and whenever user enters a new search term user is scrolled to top again.

**Screenshots/Recordings**
*before*
https://github.com/lcrojano/Giphy_Explorer/assets/113845853/a47a2d7e-227d-4e35-a16f-5c301e46b862

*after*
https://github.com/lcrojano/Giphy_Explorer/assets/113845853/2f3ae590-354a-485c-96b9-fcbc4ea1da81

Please review my pr , if i can make any changes .I think this looks good. 
This will help in my hactoberfest contribution.Thank you🥰✌️

### CODE REVIEW

**Comment by lcrojano** [comment]
Your approach is good, and we can improve it a little. It would be nice if the search bar moved to the top of the page with a small margin (20px, for example) when the user scrolls down. It seems that your current solution has more margin from the top, so it would be good to adjust it to make the bar stick to the top when the user scrolls.

image

File: styles/styles.css [hunk 1]
```diff
@@ -10,23 +11,24 @@
     --principal-text-color: white;
     --primary-text-color: #121212;
 }
+
 #myBtn {
```
Comment (lcrojano) [inline styles/styles.css:15]: You can add negative margin to keep the search bar closer to the top while scrolling.

**Review by lcrojano — CHANGES_REQUESTED** [review]

**Comment by veerendra19codes** [comment]
Can make search bar on top of GIFY SEARCH and the login button so that it would be easier to stick the searchbar at top with margin top 20px.

**Comment by veerendra19codes** [comment]
Actually its midnight here ,i m going to sleep i will work on this within next 24 hrs give me some time tomorrow i have a busy schedule.I got what you are saying.Just give me some time.Thank you for cooperating😅🙌

**Review by veerendra19codes — COMMENTED** [review]
Made the changes you said, fixed searchbar at the top with 20px gap above, also added placeholder.How does it looks now😅😅

**Screenshots/Recordings**
*after*
https://github.com/lcrojano/Giphy_Explorer/assets/113845853/9450bdf9-fef9-4023-b31a-896d8f1be725

@lcrojano Please review this commit🥰✌️

**Comment by veerendra19codes** [comment]
Thank you for merging🥰🥰

**Pull request merged**